jp.nonbili.noutube.yml

```diff
@@ -0,0 +1,37 @@
+app-id: jp.nonbili.noutube
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '25.08'
+command: run.sh
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
+  - --talk-name=org.freedesktop.PowerManagement
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.gnome.SettingsDaemon
+modules:
+  - name: noutube
+    buildsystem: simple
+    build-commands:
+      - cp -a linux*unpacked /app/main
+      - install -Dm755 -t /app/bin/ ./run.sh
+
+      - install -Dm644 linux-unpacked/flatpak/jp.nonbili.noutube.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 linux-unpacked/flatpak/jp.nonbili.noutube.desktop -t /app/share/applications/
+      - install -Dm644 linux-unpacked/resources/app.asar.unpacked/resources/icon.png -T /app/share/icons/hicolor/512x512/apps/jp.nonbili.noutube.png
+    sources:
+      - type: archive
+        url: https://github.com/nonbili/NouTube-Desktop/releases/download/v0.1.9/linux-unpacked.zip
+        sha256: b05ebbbe43c7c1d76b55d3e980e93b3895830fb49b3d5f8ef8d24410a8f0549a
```

Comment:
` <project_license>AGPL-3.0-or-later</project_license>`

so https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source

Reply:
and ignore aarch64?

Reply:
May I ask for an exception here, I'm basically copying the behavior of freetube https://github.com/flathub/io.freetubeapp.FreeTube/blob/9a93e19a23add424559fe3f5bc388cbb15ad040f/io.freetubeapp.FreeTube.yml#L29

I did find an example of using bun #7078, it requires me uploading the whole node_modules folder to github release page though...

Reply:
<img width="1244" height="76" alt="image" src="https://github.com/user-attachments/assets/16e9eee2-c7ea-45d2-8578-70eab4d9178b" />
because it's a react-native/electron universal repo, the node_modules folder is too big